### DISCUSSION TITLE
Add tradingviews lightweight-charts library

### DISCUSSION THREAD

**Discussion by 1081** [zauberzeug/nicegui#813] [Ideas / Feature Requests]
Would it be possible to add TradingView lightweight-charts library (open source) in a similar way to Highcharts? 
The library is perfect for creating interactive financial charts and is designed to be lightweight and fast, making it a great fit for NiceGUI. I personally believe that this could have a positive impact on expanding the user base.

https://www.tradingview.com/lightweight-charts/

**Comment by AstraDynamic** [comment]
This would indeed be very helpful. It has some advantages over highcharts and is outrageously fast.

**Comment by falkoschindler** [comment]
That would be nice! Maybe someone draft an implementation based on `ui.charts` and create a pull request?

**Reply by srinivasb93** [reply]
This would be a nice add to nicegui. Can someone look at it please?

**Reply by srinivasb93** [reply]
@falkoschindler , Would like to check if this can be considered for implementation in the near future please?

**Reply by falkoschindler** [reply]
@srinivasb93 Unfortunately I can't promise anything at the moment. There are quite a lot of issues and PRs waiting for our attention before we can tackle new features. According to the number of upvotes many people are interested in this feature though. But to speed things up any help is greatly appreciated.

**Reply by azjps** [reply]
fyi @srinivasb93 I shared a simple example embedding a lightweight chart in https://github.com/zauberzeug/nicegui/pull/5137
